bridge/README.md

```diff
@@ -0,0 +1,33 @@
+# Bitwarden Native Bridge SDK
+
+## Contents
+
+- [Compatibility](#compatibility)
+- [Versioning](#versioning)
+
+## Compatibility
+
+- **Minimum SDK**: 28
+- **Target SDK**: 34
+
+## Versioning
+This repository conforms to the following versioning convention:
+
+**v[MAJOR].[MINOR].[PATCH]**
+
+```
+where [RELEASE]   is incremented to represent major milestones that indicate a significant change in the library.
+
+      [MINOR]  is incremented when any standard changes (breaking or otherwise) are introduced to the library.
+
+      [PATCH]     is incremented when a hot-fix patch is required to an existing minor
+                  release due to a bug introduced in that release.
+```
+
+### Specification Highlights: ###
+
+Some specifications & FAQ that have come up when using this system.
+
+1. All updates should have a corresponding `CHANGELOG.md` entry that at a high-level describes what is being newly introduced in it.
```

Comment:
Are we going to add the initial changelog and docs describing how it works later?

Reply:
I'd like to add it now. Gonna push it up in a sec 👍